### PR TITLE
VEGA-2059 add reduced fee approved event

### PR DESCRIPTION
**Pull request by GemTay**
VEGA-2059 #patch

### CODE REVIEW

File: domains/POAS/events/reduced-fee-approved/examples/example.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "uid": "M-1234-5678-9012",
+  "outstandingFee": 4100
```
Comment (hawx) [inline domains/POAS/events/reduced-fee-approved/examples/example.json:3]: Could it be called `fee` or even `feePence`? The "outstanding" makes it sound like it is calculating what is left to pay

Comment (GemTay) [reply domains/POAS/events/reduced-fee-approved/examples/example.json:3]: Ah sorry that's what I thought we wanted but I must have misunderstood. So do want the amount to be the new expected total to pay? E.g A remission approved is £41, but a remission or exemption declined is £82?

Comment (hawx) [reply domains/POAS/events/reduced-fee-approved/examples/example.json:3]: yeah that will be simpler, we have the amount in MRLPA of what has been paid so can then do `fee - paid = outstanding`

File: domains/POAS/events/reduced-fee-approved/schema.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "type": "object",
+  "properties": {
+    "uid": {
+      "type": "string",
+      "description": "The UID of the LPA",
+      "pattern": "M(-[A-Z0-9]{4}){3}"
+    },
+    "fee": {
+      "type": "integer",
+      "description": "The expected total to be paid as a result of the fee reduction"
```
Comment (andrewpearce-digital) [inline domains/POAS/events/reduced-fee-approved/schema.json:14]:
```suggestion
      "description": "The expected total to be paid in pence as a result of the fee reduction"
```

**Review by hawx — APPROVED** [review]

**Pull request merged**